server/src/live-pricing.js

```diff
@@ -71,24 +75,24 @@ const createSession = (params) => {
 
 const pollSuccess = (state, data) => {
   if (state.finished) {
-    return;
+    return null;
   }
 
   if (data.Status === 'UpdatesComplete' || state.timedOut) {
     console.log('polling complete');
-    state.finished = true;
+    state.finished = true; // eslint-disable-line no-param-reassign
```

Comment:
Decided to ignore this as untangling all this state passing around looked like it would be quite big. We can do it if people disagree though.